homeassistant/components/switch/hikvisioncam.py

```diff
@@ -15,7 +15,7 @@
 from homeassistant.helpers.entity import ToggleEntity
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['hikvision==1.2']
+REQUIREMENTS = ['hikvision==0.4']
```

Comment:
Maybe add a comment that this is the latest version known to be working. That could save us from repeating this mistake.

Reply:
Done!